packages/ui-toolkit/src/components/atoms/Toaster/assets.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+import { ToastTypes } from './types';
+
+export const getAsset = (type: ToastTypes): JSX.Element | null => {
+  switch (type) {
+    case 'success':
+      return SuccessIcon;
+
+    case 'info':
+      return InfoIcon;
+
+    case 'warning':
+      return WarningIcon;
+
+    case 'error':
+      return ErrorIcon;
+
+    case 'default':
+
+    default:
+      return null;
+  }
+};
+
+
+const SuccessIcon = (
```

Comment:
Can we use `mint-icons` here ?

Reply:
@saloni-groww 
I have used the same icons used in existing toast component to maintain consistency in UI, 
for changing icons we'll also needed UX testing and design sign off. can be solved after design discussion.

Reply:
@ShivshankarPindoriya Can we prioritise this in this PR only ?

Reply:
as discussed we can plan it after migration as well as it will be time taking task.